plot-doc/plot/scribblings/ticks.scrbl

```diff
@@ -606,11 +606,31 @@ Represents a tick with a label.
 }
 
 @defthing[ticks-layout/c contract? #:value (-> real? real? (listof pre-tick?))]{
-The contract for tick layout functions. Note that a layout function returns @racket[pre-tick]s, or unlabeled ticks.
+
+  The contract for tick layout functions in @racket[ticks] structures.  The
+  function receives axis bounds and returns a list of @racket[pre-tick]s to be
+  shown on the axis.
+
+  Note that the layout function returns @racket[pre-tick]s, or unlabeled
+  ticks, and a separate format function is used to produce the labels for the
+  ticks.
+
 }
 
 @defthing[ticks-format/c contract? #:value (-> real? real? (listof pre-tick?) (listof string?))]{
-The contract for tick format functions. A format function receives axis bounds so it can determine how many decimal digits to display (usually by applying @racket[digits-for-range] to the bounds).
+
+  The contract for tick format functions in @racket[ticks] structures.  The
+  format function receives axis bounds and a list of @racket[pre-tick]s and
+  return a label for each @racket[pre-tick] in this list.
+
+  The returned labels should be usually distinct, as the plot library will
+  consider ticks with the same label to be duplicates and collapse them,
```

Comment:
Is "same label" determined by `string=?`? If so is that worth mentioning?

Reply:
Yes it is.  I added a clarrification.